jupyterlab_comments/jupyterlab_comments/git_commands.py

```diff
@@ -0,0 +1,71 @@
+import subprocess
+import json
+from traitlets.config.configurable import Configurable
+from traitlets import Int, Float, Unicode, Bool
+
+
+class Git(Configurable):
+    """
+    Remote repository should be configured by the
+    user in their Jupyter config file. Default remote is 'origin'
+    """
+    remote = Unicode(u'origin', config=True)
+
+    def run(self, *args):
+        try:
+            return subprocess.check_output(['git'] + list(args))
+        except subprocess.CalledProcessError as e:
+            print("Error invoking git command")
+            print(e.output)
+
+    def status(self):
+        status = self.run('status').decode('utf-8')
+        return status
+
+    def appraise_pull(self):
+        self.run('appraise', 'pull', self.remote)
+
+    # Return true if the current directory is a git repository
+    def inside_git_repo(self):
```

Comment:
Right now, I believe this just tells you whether or not Jupyter was started from within a git repository, not whether or not the user is navigating within a git repository.

It needs to set the `cwd` keyword argument similarly to the `run` method above.